watch-parties/src/main/webapp/style.css

```diff
@@ -0,0 +1,37 @@
+div{
+    text-align: center;
+}
+
+#chat {
+    width: 200px;
+    float: right;
+    align-content: left;
+    align-self: right;
+}
+
+#chatlog {
+    height: 1152px;
```

Comment:
This should just be the height of the page (`100vh`) or the container (`100%`) instead of a specific size.